content/en/apps/examples/covid-education.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "COVID-19 Education and Training for CHWs"
+linkTitle: "COVID-19 Education"
+weight: 
+description: >
+ Education and training for Community Health Workers to combat COVID-19.
+keywords: training onboarding 
+relatedContent: >
+  apps/examples/training
+  apps/examples/phc-covid
+  apps/concepts/access/#magic-links-for-logging-in
+---
+
+The COVID-19 pandemic has created unique challenges to providing in-person Community Health Worker training. To support CHWs, three learning modules were created to rapidly and remotely train them on COVID-19 safety protocols and patient care. Through CHT app and SMS deployments, Supervisors can now train CHWs on care workflows without being physically present. This example covers the following learning modules:
+
+* Health safety protocols for preventing the spread of COVID-19
+* Preventing the dissemination of misinformation about COVID-19
+* Recognizing COVID-19 and caring for patients with suspected infection
+
+## Problem Being Addressed
+
+Now, more than ever, it is important for Community Health Workers to stay safe and serve their communities by understanding effective health protocols for preventing the spread of COVID-19. Training would traditionally be done in person, however, due to public health concerns, travel and gatherings of people are limited. Health systems have been forced to quickly adapt to deliver training content in a safely distanced, timely, and effective manner.  
+
+## Solution Overview
+
+The CHT provides a flexible way to deliver timely COVID-19 remote training and education to CHWs. In collaboration with [Stanford's Digital Medic Team](https://digitalmedic.stanford.edu/), and vetted by a team of health experts at Stanford, a form-based application was designed to deliver three remote learning modules. Utilizing tasks to order the modules, CHWs can progress through training at their own pace. The training modules:
```

Comment:
I think "form-based" will be confusing in the context of the doc site, reworded accordingly:

```suggestion
The CHT provides a flexible way to deliver timely COVID-19 remote training and education to CHWs. In collaboration with [Stanford's Digital Medic Team](https://digitalmedic.stanford.edu/), and vetted by a team of health experts at Stanford, three remote learning modules were designed. CHWs can progress through training at their own pace, and receive tasks in their CHT app to encourage them to complete the modules. The training modules:
```